hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -59,15 +59,18 @@ export default function HackathonDetails(props: IProps) {
   const [hackathonData, setHackathonData] = useState({} as Hackathon)
   const [ideaData, setIdeaData] = useState<Idea>()
   const [relevantIdeaList, setRelevantIdeaList] = useState([] as Idea[])
-  const [value, onChange] = useState(hackathonData.description)
   const [votingOpened, setVotingOpened] = useState<boolean>(false)
 
+  const initialValue =
+    '<p>Please add your <b>hackathon description</b> here</p>'
+  const [descriptionValue, onChange] = useState(initialValue)
```

Comment:
it actually looks like you never use this value anywhere 🤔

you set it whenever someone types in the RichTextEditor but I don't see it being access anywhere

Reply:
ah, because it is read only... ok... then

I don't think you need this tbh, you should really just be getting the value from the hackathon which is  provided to this class.  In addition you pass in a `onChange` listener to the RichTextEditor which it probably doesn't need at all (since it is not editable)

The backend also is meant to ensure that a description is never null so you shouldn't actually need the checks that you have everywhere (`xxx || ''`), you should just construct a "valid" hackathon in the place where you make the state hook